incor/main.py

```diff
@@ -65,6 +75,8 @@ def get_arg(arg, default=None):
     else:
         sys.argv = sys.argv[1:]
         path = " ".join(sys.argv) if len(sys.argv) > 0 else '.'
+        if str(oct(os.stat('file').st_mode)[3:]) not in ("755","777","455"):
```

Comment:
What exactly do you refer to with 'file' ? 
Please Do Install the code in develop mode, run it and check if everything is working before sending a pull request. 
I am getting "No such file or directory error: 'file' "
Resolve it and make a pull request.